can/io/player.py

```diff
@@ -6,8 +6,11 @@
 in the recorded order an time intervals.
 """
 
+import pathlib
 from time import time, sleep
-import logging
+import typing
+
+import can  # pylint: disable=unused-import
```

Comment:
If this is for type-checking only, you can do something like:

```python
if typing.TYPE_CHECKING:
    import can
```

Reply:
I was not able to do that because then sphinx complained about not knowing some targets (i.e. classes).

Reply:
Okay, it worked.